src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.datasource.url=jdbc:neo4j:bolt://localhost
+spring.datasource.driver-class-name=org.neo4j.jdbc.bolt.BoltDriver
+spring.datasource.username=neo4j
+spring.datasource.password=toto
```

Comment:
True, I actually stumbled upon that page quite a few times in the last couple of weeks :-)